src/app/app.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
```

Comment:
issue

          declarations: [AppComponent],

          Explantion: AppComponent should be declared in declarations instead of imports for the test module.